spec/test-utils/test-data/generate-test-data.py

```diff
@@ -72,6 +72,30 @@ def main() -> None:
         master_private_key.public_key().public_bytes(Encoding.Raw, PublicFormat.Raw)
     )
 
+    b64_master_private_key = encode_base64(
+        master_private_key.private_bytes_raw()
+    )
```

Comment:
is this the same as `encode_base64(MASTER_CROSS_SIGNING_PRIVATE_KEY_BYTES)`? if so, that might be clearer